src/worker.cc

```diff
@@ -598,16 +601,17 @@ void WorkerThread::ProcessError(struct ErrorRecord *error, int priority,
   error->vbyteaddr = reinterpret_cast<char *>(error->vaddr) + offset;
 
   // Find physical address if possible.
-  error->paddr = os_->VirtualToPhysical(error->vbyteaddr);
+  error->paddr = os_->VirtualToPhysical(error->vbyteaddr, *test_step_);
 
   // Pretty print DIMM mapping if available.
   os_->FindDimm(error->paddr, dimm_string, sizeof(dimm_string));
 
   // Report parseable error.
   if (priority < 5) {
     // Run miscompare error through diagnoser for logging and reporting.
-    os_->error_diagnoser_->AddMiscompareError(
-        dimm_string, reinterpret_cast<uint64>(error->vaddr), 1);
+    // TODO(b/275900374) Make this a diagnosis
+    // os_->error_diagnoser_->AddMiscompareError(
+    //     dimm_string, reinterpret_cast<uint64>(error->vaddr), 1);
```

Comment:
ditto

Reply:
I wanted to keep that so that I don't lose track of what the code was doing before until I'm ready to translate these logs as part of the attached bug.